index.html

```diff
@@ -884,6 +857,34 @@ <h2 id="deviceId">
         </div>
       </section>
     </section>
+    <section>
+      <h1 id=feature-policy-integration>Feature Policy Integration</h1>
+      <p>This specification defines a <a data-link-type="dfn"
+        href="https://wicg.github.io/feature-policy/#policy-controlled-feature"
+        id="ref-for-policy-controlled-feature">policy-controlled feature</a>
+      identified by the string
+      <code>"<dfn data-lt="display-feature">display</dfn>"</code>. It's
+      <a data-link-type="dfn"
+        href="https://wicg.github.io/feature-policy/#default-allowlist"
+        id="ref-for-default-allowlist">default allowlist</a> is <code>"self"</code>.
+      <div class="note">
+        <p>A <a data-link-type="dfn"
+          href="https://dom.spec.whatwg.org/#concept-document"
+          id="ref-for-concept-document①⓪">document</a>'s
+        <a data-link-type="dfn"
+          href="https://html.spec.whatwg.org/multipage/dom.html#concept-document-feature-policy"
+          id="ref-for-concept-document-feature-policy">feature policy</a>
+        determines whether any content in that document is allowed to use
+        <code>getDisplayMedia</code>. If disabled in any document, no content in
+        the document will be
+        <a data-link-type="dfn"
+          href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#allowed-to-use"
+          id="ref-for-allowed-to-use②">allowed to use</a>
```

Comment:
Is the circled 2 on this line an accidental character?

Reply:
Boiler-plate from the fullscreen spec I copied. Removed now. Thanks for spotting it!